LIPs/LIP-inflation-params.md

```diff
@@ -0,0 +1,96 @@
+
+Note, that while the above are suggested values, the community should debate these values for inclusion in the final LIP proposal. It has also been considered setting the `inflationCeiling` to something below the current rate, such as 250,000, representing 10% per annum.
+
+## Rationale
+
+Bounding the inflation rate provides several benefits:
+
+- **Predictability**: Stakeholders can anticipate reward rates, aiding in decision-making.
+- **Economic Stability**: Prevents runaway inflation or deflation, making it reasonable to predict the value of an individual token which is important when considering staking decisions and public goods funding governance.
+- **Alignment with Network Growth**: Ensures that inflation rates reflect actual network participation and activity levels.
+
+There are currently two schools of thought on setting the initial `inflationCeiling`.
+
+- **Option 1 - Set the ceiling above the current inflation level, such as 30% annually**: The argument here is to let the current economic incentives play out, while putting a cap to prevent runaway inflation. Enacting this change would have no impact on the short term rewards issued by the protocol as they'd continue to rise until the target participation rate was exceeded. Allowing the parameter to be used and adjusted in the future, gives more governance control to the community, who could always decide to change it later.
+- **Option 2 - Set the ceiling to a value below the current inflation level, such as 10% annually** - The argument here is to begin to drive down inflation over time to a rate that is more in line with broader market expectations, DePIN norms, and immediately begin mitigating the potential negative effects of high inflation. This could have the side effect of causing delegators to drop out as rewards would begin decreasing immediately - although the decrease would be very slow with the current inflationChange rate. 
+
+The proposed parameter values align with observed trends in similar stake-based ecosystems, balancing incentives with sustainability.
+
+
+## Backwards Compatibility
+
+This proposal introduces new parameters to the inflation adjustment algorithm and comes with slight alterations to the existing functionality to calculate inflation. The changes are designed to be backward-compatible, from a participation perspective, ensuring seamless integration with the current protocol that don't require any client changes in terms of protocol function signatures or capabilities.
```

Comment:
Worth mentioning that the upgrade would involve a new Minter contract so protocol clients need to be aware of the new address either through a lookup through the Controller or by updating the address that they are using - otherwise they would get outdated values.